### PR TITLE
Update dashboard style to be closer to mockups

### PR DESCRIPTION
**Pull request by kisantia**
- Update look of left nav
- Dashboard widget wrapper: get rid of grey header, bold title
- Update border lines of toolbar and properties widget to change color with the theme

Mockup: https://www.figma.com/proto/QRkuP8YG7sZr87LRoS210ldo/Azure-Data-Studio?node-id=7109%3A173389&viewport=-833%2C-19569%2C0.5569696426391602&scaling=min-zoom

Changes in this PR:
image

image

horizontal tabs are still the old style:
image

### CODE REVIEW

File: src/vs/workbench/common/theme.ts [hunk 1]
```diff
@@ -144,6 +144,12 @@ export const TAB_UNFOCUSED_INACTIVE_FOREGROUND = registerColor('tab.unfocusedIna
 	hc: Color.white
 }, nls.localize('tabUnfocusedInactiveForeground', "Inactive tab foreground color in an unfocused group. Tabs are the containers for editors in the editor area. Multiple tabs can be opened in one editor group. There can be multiple editor groups."));
 
+export const DASHBOARD_TAB_ACTIVE_BACKGROUND = registerColor('tab.dashboardActiveBackground', {
+	dark: '#e1f0fe',
```
Comment (kisantia) [inline src/vs/workbench/common/theme.ts]: waiting to hear back from Raden on what the color here should be for dark theme

Comment (alanrenmsft) [reply src/vs/workbench/common/theme.ts]: 😆

Comment (alanrenmsft) [reply src/vs/workbench/common/theme.ts]: @kisantia how about copy the style of portal?

**Comment by udeeshagautam** [comment]
the extension icons doesnt seem to show in dark theme - is that a snip issue or actual thing
image

**Comment by alanrenmsft** [comment]
> the extension icons doesnt seem to show in dark theme - is that a snip issue or actual thing
> image

I'll take a look at this one.

File: src/vs/workbench/common/theme.ts (hunk 1 above)
Comment (kisantia) [reply src/vs/workbench/common/theme.ts]: updated with values from Raden

File: src/sql/workbench/contrib/dashboard/browser/contents/dashboardWidgetWrapper.component.ts [hunk 2]
```diff
@@ -220,7 +220,7 @@ export class DashboardWidgetWrapper extends AngularDisposable implements OnInit
 	private updateTheme(theme: IColorTheme): void {
 		const el = <HTMLElement>this._ref.nativeElement;
 		const headerEl: HTMLElement = this.header.nativeElement;
-		let borderColor = theme.getColor(themeColors.SIDE_BAR_BACKGROUND, true);
```
Comment (alanrenmsft) [inline src/sql/workbench/contrib/dashboard/browser/contents/dashboardWidgetWrapper.component.ts]: what is the 2nd parameter for? did you mean to remove it?

Comment (kisantia) [reply src/sql/workbench/contrib/dashboard/browser/contents/dashboardWidgetWrapper.component.ts]: Setting it to true doesn't do anything: "@param useDefault specifies if the default color should be used. If not set, the default is used."

**Pull request merged**